src/main/java/com/example/backendspringboottechiteasycontrollerles10/services/TelevisionService.java

```diff
@@ -0,0 +1,168 @@
+    }
+
+    public TelevisionDto updateTelevision(@PathVariable Long id, @RequestBody TelevisionDto updateTelevision) {
+        Optional<Television> television = televisionRepository.findById(id);
+        if (television.isEmpty()) {
+            throw new RecordNotFoundException("Er is geen televisie met id nummer: " + id);
+        } else {
+            Television television1 = television.get();
+            television1.setType(updateTelevision.getType());
+            television1.setBrand(updateTelevision.getBrand());
+            television1.setName(updateTelevision.getName());
+            television1.setPrice(updateTelevision.getPrice());
+            television1.setAvailableSize(updateTelevision.getAvailableSize());
+            television1.setRefreshRate(updateTelevision.getRefreshRate());
+            television1.setScreenType(updateTelevision.getScreenType());
+            television1.setScreenQuality(updateTelevision.getScreenQuality());
+            television1.setSmartTv(updateTelevision.getSmartTv());
+            television1.setWifi(updateTelevision.getWifi());
+            television1.setVoiceControl(updateTelevision.getVoiceControl());
+            television1.setHdr(updateTelevision.getHdr());
+            television1.setBluetooth(updateTelevision.getBluetooth());
+            television1.setAmbiLight(updateTelevision.getAmbiLight());
+            television1.setOriginalStock(updateTelevision.getOriginalStock());
+            television1.setSold(updateTelevision.getSold());
+
+            Television television2 = televisionRepository.save(television1);
+            return fromTelevision(television2);
+        }
+    }
+
+    // stap 13 t/m 16 - maar de functie werkte niet bij aanroepen wanneer deze zich in de TelevisionDto.java bevond dus heb hem hierheen verplaatst.
+    public TelevisionDto fromTelevision(Television television) {
+        var televisionDto = new TelevisionDto();
+        televisionDto.setId(television.getId());
+        televisionDto.setType(television.getType());
+        televisionDto.setBrand(television.getBrand());
+        televisionDto.setName(television.getName());
+        televisionDto.setPrice(television.getPrice());
+        televisionDto.setAvailableSize(television.getAvailableSize());
+        televisionDto.setRefreshRate(television.getRefreshRate());
+        televisionDto.setScreenType(television.getScreenType());
+        televisionDto.setScreenQuality(television.getScreenQuality());
+        televisionDto.setSmartTv(television.getSmartTv());
+        televisionDto.setWifi(television.getWifi());
+        televisionDto.setVoiceControl(television.getVoiceControl());
+        televisionDto.setHdr(television.getHdr());
+        televisionDto.setBluetooth(television.getBluetooth());
+        televisionDto.setAmbiLight(television.getAmbiLight());
+        televisionDto.setOriginalStock(television.getOriginalStock());
+        televisionDto.setSold(television.getSold());
+
```

Comment:
Je hebt in deze methode nog niet je relaties gezet. Wanneer je dus een television opvraagt via een get request, krijg je een TelevisionDto terug met daarin null-waardes voor RemoteController en CiModule.